File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/Entities/Court.cs
namespace PickleballCourtBookingSystem.Core.Entities

{
    public class Court
    {
        public Guid? Id { get; set; }
        public int? CourtNumber { get; set; }
        public string? Description { get; set; }
        public Guid? CourtClusterId { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/Court.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Court(
    var id: String,
    var courtNumber: Int,
    var description: String?,
    var courtClusterId: String
) : Parcelable